app/build.gradle

```diff
@@ -115,6 +115,9 @@ dependencies {
     // Dagger
     implementation "com.google.dagger:dagger:$dagger_version"
     implementation "com.google.dagger:dagger-android-support:$dagger_version"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
We haven't dicussed this. The idea is to move version number to an ext var, such as dagger or other dependencies

Reply:
✔ **Done**